Keybase v6.4.0 patch (#6)
RalfJung/io.keybase.Keybase: README.md

```

Now you can run it like any other flatpak.

## Updating

> This respository provides flatpak for a fixed version of keybase.
> See [here](https://github.com/RalfJung/io.keybase.Keybase/blob/master/io.keybase.Keybase.yml#L54) for the current version.
> 
> To update keybase flatpak you need up date this line (see comment for where to find latest release file name), and then rebuild and install with the command above.
>  This should preserve your settings & database.

Reviewer: ```suggestion
This repository provides a flatpak for a fixed version of keybase.
See [here](https://github.com/RalfJung/io.keybase.Keybase/blob/master/io.keybase.Keybase.yml#L54) for the current version.

To update the keybase version, you need up date this line (see comment for where to find latest release file name), and then rebuild and install with the command above.
This should preserve your settings & database.
```

```

Now you can run it like any other flatpak.

## Updating

> This respository provides flatpak for a fixed version of keybase. See [here](https://github.com/RalfJung/io.keybase.Keybase/blob/master/io.keybase.Keybase.yml#L54) for current version.

Reviewer: ```suggestion
This respository provides flatpak for a fixed version of keybase.
See [here](https://github.com/RalfJung/io.keybase.Keybase/blob/master/io.keybase.Keybase.yml#L54) for the current version.
```

> To update keybase flatpak you need up date this line (see comment for where to find latest release file name), and then rebuild and install with command above. This should preseve you settings & database.

Reviewer: ```suggestion
To update keybase flatpak you need up date this line (see comment for where to find latest release file name), and then rebuild and install with the command above.
 This should preserve your settings & database.
```
